app/controllers/instances_controller.rb

```diff
@@ -29,6 +29,8 @@ def show # Display the game instance with users subscribed (web socket)
     @game = Game.find(@instance.game_id)
     @host = User.find(@instance.user_id)
     @players = Player.where(instance_id: @instance.id)
+    # @is_host = check_host(@instance)
+
```

Comment:
items with note to self are the lines of code I've added that are not needed. I'll clean them up on a separate PR. Will just have to add this note for quick reference.

note to self: cleanup